.gitignore

```diff
@@ -4,4 +4,4 @@ local.properties
 
 /.idea
 /.gradle
-build
+gradle
```

Comment:
I don’t think this should be added back. Merge conflict maybe?

Reply:
Ah yes I mis-corrected that merge conflict, I'll remove this too.